src/openrct2/rct1/S4Importer.cpp

```diff
@@ -2834,8 +2834,8 @@ namespace OpenRCT2::RCT1
         dst->NumLaps = src->NumLaps;
         dst->var_D3 = src->VarD3;
         dst->scream_sound_id = OpenRCT2::Audio::SoundId::Null;
-        dst->pitch = src->Pitch;
-        dst->roll = src->BankRotation;
+        dst->pitch = static_cast<VehiclePitch>(src->Pitch);
+        dst->roll = static_cast<VehicleRoll>(src->BankRotation);
```

Comment:
Can we change the fields in S4/S6 to use `VehiclePitch` and `VehicleRoll` instead?